src/components/logic/operate.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import Big from 'big.js';
```

Comment:
- [ ] `Required`
- [ ] Please enable the disabled eslint because it defeats the purpose of using linters. 🥂